Resources/public/js/views/ez-testonearticlesview.js

```diff
@@ -0,0 +1,195 @@
+YUI().add('ez-testonearticlesview', function(Y) {
+    'use strict';
+    /**
+     * Provides the article list subview of test one
+     *
+     * @module ez-testonearticlesview
+     */
+    Y.namespace('eZ');
+
+    var ARTICLE_LINK_SELECTOR = '[data-location-id="{locationId}"]',
+        ARTICLES_CLASS_PREFIX = 'ez-view-testonearticlesview',
+        LOADER = Y.Node.create('<span class="ez-inline-loader"></span>');
+
+    /**
+     * The article list view
+     *
+     * @namespace eZ
+     * @class TestOneArticlesView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.TestOneArticlesView = Y.Base.create('testOneArticlesView', Y.eZ.TemplateBasedView, [], {
+        initializer: function () {
+            this.after('articlesChange', this.render);
+            this.on('rootLocationChange', this._changeRootLocation);
+            this.on('*:articleLoading', this._articleLoading);
+            this.on('*:articleLoaded', this._articleLoaded);
+        },
+
+        render: function () {
+            var history = this.get('history') || [],
+                currentLocation = this.get('currentLocation');
+
+            this.get('container').setHTML(this.template({
+                articles: this.get('articles'),
+                previous: history.length > 0 ? history[history.length - 1].article : null,
+                currentLocation: currentLocation ? currentLocation.name : null
+            }));
+
+            this.get('container').all('[data-location-id]').on('click', this._selectArticle, this);
```

Comment:
Referencje do kontenera dobrze jest trzymać w zmiennej, aby nie wywoływać `this.get('container')` po kilka razy.